apps/pipelines/nodes/nodes.py

```diff
@@ -1110,10 +1111,20 @@ def _process(self, input: str, state: PipelineState) -> PipelineState:
         kwargs = {}
         try:
             exec(byte_code, custom_globals, custom_locals)
-            result = str(custom_locals[function_name](input, **kwargs))
+            result = custom_locals[function_name](input, **kwargs)
+        except WaitForNextInput:
```

Comment:
What is the effect of this on the pipeline's execution? Will it end the pipeline?